canonical/lxd-ui
# Generalize config table helpers WD-7725

## Done

- generalized the config table helpers, and improved naming for all forms with inheritance behaviour

Fixes WD-7725

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check create and edit forms for
      - instances
      - profiles
      - projects
      - storage volumes
      - networks
    - ensure the test suite passes
    - no functional changes should be visible
